airbyte-webapp/src/core/resources/Connection.ts

```diff
@@ -46,6 +46,7 @@ export default class ConnectionResource extends BaseResource
   readonly destinationId: string = "";
   readonly syncMode: string = "";
   readonly status: string = "";
+  readonly message: string = "";
```

Comment:
Are you sure we need `message` prop here? I can't see it in `Connection` interface and now quite sure that it is used anywhere.

Reply:
airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/SettingsView.ts#54